islandora_compound_object.module

```diff
@@ -748,7 +748,7 @@ function islandora_compound_object_retrieve_compound_info(AbstractObject $object
     }
 
     $info = array(
-      'parent_label' => $parent->label,
+      'parent_label' => ($parent && $parent->label) ? $parent->label : NULL,
```

Comment:
oh, wait. I approved this too soon? Do you want to show the object label even if access is not possible? Just a question, not sure how we deal with this in other parts.

Reply:
In my use case, the parent object should be ingested eventually as part of the batch ingest. Otherwise, I believe it is treated as an empty string. I can make it a default string, say "Parent Unavailable", to avoid any other potential problems. 
Thoughts?

Reply:
My question is really in the case of "not accesible because of permissions" but still there. Not the use case you are trying to fix really. Means: do we show object labels for objects the user can not access? This goes beyond the is there or not i guess

Reply:
I am not sure how valuable it would be to hide the name. 

I see that blocks such as Islandora Compound Object Navigation and Islandora Compound Object JAIL Display are already permission-able and so this case would fall to the responsibility of the user/admin. 
Maybe that is an edge case that doesn't need to be addressed?

Reply:
Ok, lets go this way then.